### PR TITLE
Rewrite user rating UI

### PR DESCRIPTION
**Pull request by artur-masalcev**
Rating UI is fully rewritten to match clean code principles. Also big styling changes were made.

### CODE REVIEW

**Comment by artur-masalcev** [comment]
> Overall rating does not update after giving a rating

Fair point. I'll make separate styles for both of these buttons

**Comment by Mantas330BJ** [comment]
> > Overall rating does not update after giving a rating
> 
> And it shouldn't. I've only implemented UI part of rating system and also added functions templates for further functionality. Since the previous functionality was implemented by directly changing attributes of the view (and not actual data) I removed it, because it will be changed anyway sooner or later

Got it

File: Wasted/FoodPlaceRatingSystem/RatingPopup.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+        public String FoodPlaceTitle { get; }
+        public int FoodPlaceID { get; }
+        public int Rating
+        {
+            get
+            {
+                return ratingBar.SelectedStarValue;
+            }
+        }
```
Comment (Mantas330BJ) [inline Wasted/FoodPlaceRatingSystem/RatingPopup.xaml.cs:19]: Readonly properties look nicer if implemented in this way:
public int Rating => ratingBar.SelectedStarValue;

File: Wasted/FoodPlaceRatingSystem/RatingToEmoji.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        /// <param name="rating">Value of rating</param>
+        /// <returns>Emoji representation for provided 'rating'</returns>
+        public static String ConvertToEmoji(int rating)
+        {
+            if (rating <= 1) return "🤢";
+            else if (rating == 2) return "😡";
+            else if (rating == 3) return "😐";
+            else if (rating == 4) return "😋";
+            else if (rating >= 5) return "😍";
+
```
Comment (Mantas330BJ) [inline Wasted/FoodPlaceRatingSystem/RatingToEmoji.cs:23]: there is no integer value that bypasses all statements and returns ""

Comment (Mantas330BJ) [reply Wasted/FoodPlaceRatingSystem/RatingToEmoji.cs:23]: Switch statement or array of emojis is preferred. Pseudocode:
Emojis = { :d, xd, :( }
return Emojis[rating - 1]

**Comment by artur-masalcev** [comment]
> FoodPlaceTitle may not be necessary in RatingPopup

Why do you think so?

**Comment by Mantas330BJ** [comment]
> > FoodPlaceTitle may not be necessary in RatingPopup
> 
> Why do you think so?

My mistake. I did not see that it was binded in xaml file and used as a title in rating popup

**Pull request merged**